.github/workflows/build-mudlet.yml

```diff
@@ -40,11 +40,13 @@ jobs:
             qt: '5.14.2'
             deploy: 'deploy'
             run_tests: 'true'
-#           - os: windows-2019
-#             buildname: 'windows'
-#             triplet: x64-mingw-dynamic
-#             # compiler: flag not used in windows pipeline
-#             qt: '5.14.2'
+          - os: windows-2019
+            buildname: 'windows64'
+            triplet: x86_64-w64-mingw32
```

Comment:
Not sure about this and the other triplet - have these values been plucked out of the air or do they actually represent valid values to use?

Reply:
So I think the triplets aren't needed for the msys builds as they reference something vcpkg uses?